src/Subscriber/AutoUpdateTitleWithLabelSubscriber.php

```diff
@@ -76,8 +76,11 @@ public function onPullRequest(GitHubEvent $event)
             $prPrefix .= '['.$label.']';
         }
 
+        // Clean string from all HTML chars and remove whitespace at the beginning
+        $prTitle = preg_replace('@^[\h\s]+@u', '', html_entity_decode($prTitle));
+
         // Add back labels
```

Comment:
Outdated comment?

Reply:
No, I dont think so. The `$prPrefix` variables has the labels.